src/App/Routes.jsx

```diff
@@ -7,11 +7,13 @@ import HomePage from 'pages/HomePage';
 import PasswordResetPage from 'pages/PasswordResetPage';
 import UserFeedPage from 'pages/UserFeedPage';
 import NotFoundPage from 'pages/NotFoundPage';
+import Search from 'pages/SearchPage';
 
 const Routes = () => (
   <Switch>
     <Route exact path="/" component={HomePage} />
     <Route exact path="/login" component={UserFeedPage} />
+    <Route exact path="/search" component={Search} />
```

Comment:
@Bastieno Did we ask the PO if he wants the search to be on a totally different page?